pkg/auth/jwt_token.go

```diff
@@ -0,0 +1,86 @@
+package auth
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/cortezaproject/corteza-server/pkg/id"
+	//"github.com/cortezaproject/corteza-server/store"
+	"github.com/cortezaproject/corteza-server/system/types"
+	"time"
+)
+
+type (
+	ExtraReqInfo struct {
+		RemoteAddr string
+		UserAgent  string
+	}
+)
+
+func (t *token) Generate(ctx context.Context, s tokenStore, i Identifiable) (tokenString string, err error) {
```

Comment:
How about if we keep the `tokenStore` directly on the `token` struct? This will allow us to call Generate without store and keep it as lightweight as possible + no passing store to the corredor's Connect.

Would that make sense?

Reply:
Yes